leveldb-library.podspec

```diff
@@ -21,7 +21,7 @@ Pod::Spec.new do |s|
 
   s.compiler_flags = '-DOS_MACOSX', '-DLEVELDB_PLATFORM_POSIX'
 
-  s.xcconfig = {
+  s.pod_target_xcconfig = {
     'HEADER_SEARCH_PATHS' => '"${PODS_ROOT}/leveldb-library" ' +
```

Comment:
@paulb777 I'd argue this might no longer be needed but haven't tested to claim this 100%

Reply:
This is required for building because leveldb has nonpublic headers accessed relative to the root of the repository, e.g. 

https://github.com/google/leveldb/blob/47cb9e2a211e1d7157078ba7bab536beb29e56dc/db/dbformat.cc#L6